tensorflow/python/ops/math_ops.py

```diff
@@ -4857,6 +4857,42 @@ def sparse_segment_sum(
         name=name,
     )
 
+@tf_export("sparse.sampled_addmm")
+def sampled_addmm(indices,
+    values,
+    dense_shape,
```

Comment:
Still not really too sure what to do about the `dense_shape` parameter here or if it's absolutely necessary

Reply:
It's not necessary for the implementation, but I might suggest keeping it for consistency with other sparse ops.   You could use it for shape validation.

Or have your input be a `SparseTensor` and return a `SparseTensor`.